modular_skyrat/modules/clock_cult/code/structures/traps/trap.dm

```diff
@@ -0,0 +1,156 @@
+//Thing that you stick on the floor
+/obj/item/clockwork/trap_placer
+	name = "trap"
+	desc = "don't trust it"
+	icon = 'modular_skyrat/modules/clock_cult/icons/clockwork_objects.dmi'
+	w_class = WEIGHT_CLASS_HUGE
+	/// The path of the trap to make when this is set down
+	var/result_path = /obj/structure/destructible/clockwork/trap
+
+
+/obj/item/clockwork/trap_placer/attack_self(mob/user)
+	. = ..()
+	if(!IS_CLOCK(user))
+		return
+
+	for(var/obj/structure/destructible/clockwork/trap/trap in get_turf(src)) // No 50-spear instakills please
+
+		if(!istype(trap, result_path))
+			continue
+		user.balloon_alert(user, "space occupied!")
+		return
+
+	to_chat(user, span_brass("You place [src], use a <b>clockwork slab</b> to link it to other traps."))
+	var/obj/new_obj = new result_path(get_turf(src))
+	new_obj.setDir(user.dir)
+
+	qdel(src)
+
+
+//Thing you stick on the wall
+/obj/item/wallframe/clocktrap
+	name = "Clockwork trap item"
+	desc = "It's a... wait what?"
+	icon = 'modular_skyrat/modules/clock_cult/icons/clockwork_objects.dmi'
+	pixel_shift = 24
+	w_class = WEIGHT_CLASS_HUGE
+	result_path = /obj/structure/destructible/clockwork/trap
+	/// What to show the user if they are a clock cultist
+	var/clockwork_desc = "It seems to be able to be placed on walls."
```

Comment:
Probably should say that it's a trap, but I guess that's just me enjoying description that don't leave stuff too unclear.

Reply:
It's a parent item; that's all it does, there is no trap involved with this one specifically